source/connect/advanced-connection-options/authentication-connection.txt

```diff
@@ -133,6 +133,13 @@ Procedure
              settings, |compass-short| uses the same proxy to connect to both 
              the cluster and identity provider.
 
+         * - Send a nonce in the Auth Code Request
+           - Optional. Includes a random nonce as a part of the auth code 
+             request to prevent replay attacks. Enabled by default.
+
+             The nonce is an important security component. This setting should 
+             only be disabled if it is not supported by your OIDC provider.
```

Comment:
[nit]

Consider a stronger phrasing. Spitballing: "Only disable this setting if it is not supported by your OIDC provider."

Reply:
Thanks Joe, I think shortening it up and removing `should` improves it quite a bit.
I'll update this to incorporate your suggestion!